geopandas/geoseries.py

```diff
@@ -288,6 +288,33 @@ def y(self):
         """
         return _delegate_property("y", self)
 
+    @property
+    def z(self):
+        """Return the z location of point geometries in a GeoSeries
+
+        Returns
+        -------
+        pandas.Series
+
+        Examples
+        --------
+
+        >>> from shapely.geometry import Point
+        >>> s = geopandas.GeoSeries([Point(1, 1), Point(2, 2), Point(3, 3)])
+        >>> s.z
+        0    1.0
+        1    2.0
+        2    3.0
+        dtype: float64
+
+        See Also
+        --------
+
+        GeoSeries.y
```

Comment:
```suggestion
        GeoSeries.x
        GeoSeries.y
```

Can you add `GeoSeries.z` to _See also_ sections of `GeoSeries.x` and `GeoSeries.y`?